DQM/CTPPS/plugins/CTPPSRandomDQMSource.cc

```diff
@@ -0,0 +1,201 @@
+/******************************************
+ *
+ * This is a part of CTPPSDQM software.
+ * Authors:
+ *   A. Bellora (Universita' e INFN Torino)
+ *
+ *******************************************/
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "DQMServices/Core/interface/DQMEDAnalyzer.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+
+#include "CondFormats/PPSObjects/interface/CTPPSPixelIndices.h"
+#include "DataFormats/CTPPSDetId/interface/CTPPSDetId.h"
+#include "DataFormats/CTPPSDigi/interface/CTPPSPixelDigi.h"
+#include "DataFormats/CTPPSDigi/interface/CTPPSPixelDataError.h"
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelCluster.h"
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrack.h"
+#include "DataFormats/Common/interface/TriggerResults.h"
+
+#include <string>
+
+//-----------------------------------------------------------------------------
+
+class CTPPSRandomDQMSource : public DQMEDAnalyzer {
+public:
+  CTPPSRandomDQMSource(const edm::ParameterSet &ps);
+  ~CTPPSRandomDQMSource() override;
+
+protected:
```

Comment:
There is no `fillDescriptions` method. See for example
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideAboutPythonConfigFile#For_Component_developers

Reply:
Added it.